invoicing_app/models.py

```diff
@@ -36,6 +36,10 @@
 
 class Order(models.Model):
 
+    class Meta:
+        managed = True
```

Comment:
The managed attribute defaults as True, so it's not necessary to define it explicitly.